libmscore/layout.cpp

```diff
@@ -3394,7 +3394,7 @@ System* Score::collectSystem(LayoutContext& lc)
             }
 
       for (Spanner* sp : _unmanagedSpanner) {
-            if (sp->tick() >= etick || sp->tick2() <= stick)
+            if (sp->tick() >= etick || sp->tick2() < stick)
```

Comment:
may work for dashes, but has bad effects on melismas, prolonging them accross system breaks

Reply:
and there is a dash before the first syllable of a new system, if the previous system ends with it

Reply:
That wouldn't be too bad (if it were an __additional__ dash, soeor of a 'courtesy dash')

Reply:
Have you ever seen a "courtesy dash" in any score?

Reply:
No. But I've missed it, and sometimes faked it using Ctrl -
Maybe be much more of an issue in German, with our generally longer multi-syllable words?

Reply:
Yes, I have probably misunderstood the idea of these comparisons. All that I wrote about ranges holds true but this is also true for `tick` and `tick2` values for spanners so these comparisons written in this way achieve the desired effect. The real problem is that ticks values for the first lyrics dash is never properly computed since it is done in `LyricsLine::layout` which has no chance to be called because of incorrect ticks values. I guess I figured out how to resolve this issue more properly so I will update this PR soon. Though that will definitely need a review.